core/app/api/customer/groups/route.ts

```diff
@@ -0,0 +1,32 @@
+import { strict } from 'assert';
+import { NextResponse } from 'next/server';
+import { z } from 'zod';
+
+export const Customers = z
+  .object({
+    id: z.number(),
+    name: z.string(),
+  })
+  .array();
+
+export type CustomersType = z.infer<typeof Customers>;
+
+export async function GET() {
+  strict(process.env.BIGCOMMERCE_ACCESS_TOKEN, 'BIGCOMMERCE_ACCESS_TOKEN is required');
```

Comment:
`strict` isn't used in this codebase, let's change this to be consistent with environment variable use elsewhere